templates/container/BUILD.bazel.jinja

```diff
@@ -46,6 +46,7 @@ container_run_and_commit(
 	"php maintenance/install.php my_wiki admin --dbname my_wiki --pass a-nice-password --dbtype sqlite --server 'http://localhost:8080' --scriptpath '' ",
 	"chown www-data:www-data data data/*",
 	# Enable Semantic MediaWiki and then update MediaWiki to deal with the upgrade
+	"echo 'wfLoadExtension( \"SemanticMediaWiki\" );' >> LocalSettings.php",
```

Comment:
Here's a modification which will get this to register as a PHP string (right now it's treated as an unidentified constant that is converted to a string—but this is brittle behavior!).

```suggestion
	# We have to use funky PHP heredoc syntax to insert a string, given our string format
	# limitations in Bazel. :(
	"echo 'wfLoadExtension(<<<END' >> LocalSettings.php", 
	"echo 'SemanticMediaWiki' >> LocalSettings.php", 
	"echo 'END);' >> LocalSettings.php", 
```

Reply:
After this is committed we have to run `./update_versions.py` again, though.